src/sentry/api/helpers/source_map_helper.py

```diff
@@ -37,6 +37,11 @@ def source_map_debug(project, event_id, exception_idx, frame_idx):
         # already mapped
         return SourceMapDebug()
 
+    # We can't demangle node's internal modules therefore we only process
+    # user-land frames (starting with /) or those created by bundle/webpack internals.
+    if event.platform == "node" and not abs_path.startswith(("/", "app:", "webpack:")):
```

Comment:
What about windows frames?

Reply:
It should be handled by `RewriteFrames` instead imo https://github.com/getsentry/sentry-javascript/blob/develop/packages/integrations/src/rewriteframes.ts

Reply:
RewriteFrames is not part of the default integrations in node though :thinkies:

Reply:
I'd rather document this behavior tbh. It's like 0,00001% of cases, where majority comes from local envs. Running node on windows in production is almost nonexistent.